tests/EA/test_deterministic_crowding.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import numpy as np
+
+from bingo.EA.DeterministicCrowding import DeterministicCrowdingEA
+from bingo.MultipleValues import SinglePointCrossover, SinglePointMutation, MultipleValueGenerator
+from bingo.EA.SimpleEvaluation import SimpleEvaluation
+from examples.OneMaxExample import MultipleValueFitnessEvaluator
+
+@pytest.fixture
+def dc_ea(evaluation):
+    crossover = SinglePointCrossover()
+    mutation = SinglePointMutation(return_true)
+    return DeterministicCrowdingEA(evaluation, crossover, mutation, 0.2, 0.2)
+
+@pytest.fixture
+def evaluation():
+    fitness = MultipleValueFitnessEvaluator()
+    return SimpleEvaluation(fitness)
+
+@pytest.fixture
+def unfit_generator():
+    return MultipleValueGenerator(return_false, 10)
+
+@pytest.fixture
+def fit_generator():
+    return MultipleValueGenerator(return_true, 10)
+
+@pytest.fixture
+def unfit_pop(unfit_generator):
+    return [unfit_generator() for _ in range(10)]
+
+@pytest.fixture
+def fit_pop(fit_generator):
+    return [fit_generator() for _ in range(10)]
+
+def mutation_function():
+    return np.random.choice([True, False])
+
+def return_false():
+    return False
+
+def return_true():
+    return True
+
+def return_total_fitness(population, evaluation):
+    evaluation(population)
+    return sum(indv.fitness for indv in population)
+
+def test_next_gen_fitness_lower_than_parents(unfit_pop, dc_ea, evaluation):
+    parent_fitness = return_total_fitness(unfit_pop, evaluation)
```

Comment:
it looks like this function will be stochastic based on the random numbers generated in the varand. please do the following to make it deterministic:
  - set the numpy random seed at the beginning of this function
  - remove the equality in line 54 (assuming you chose a seed that makes this correct)